hardware/Drivetrain.java

```diff
@@ -185,11 +185,13 @@ else if (deltaAngle > 180)
 
         return globalAngle;
     }
-    public void strafeLeft(){
-        setPower(-0.5,-0.5,0.5,0.5);
+    public void strafeLeft(double power){
+        setPower(-power, power, power, -power);
     }
-    public void strafeRight(){
-        setPower(0.5, 0.5, -0.5, -0.5);
+
+    public void strafeRight(double power){
+        setPower(power, -power, -power, power);
+
```

Comment:
Let's test these some more but we will probably need to imtroduce some sort of PID loop into this, especially because our bot is unbalanced.